content/Certification/Guides/PICSandPIXIT.md

```diff
@@ -0,0 +1,201 @@
+| ANC.S.A0000   | Device implements ANC attribute 0x0000 on the ANC server cluster           |
+| ANC.S.A0001   | Device implements ANC attribute 0x0001 on the ANC server cluster           |
+| ANC.S.C00.Rsp | Device accepts ANC command 0x00 on the ANC server cluster                  |
+| ANC.S.C01.Rsp | Device accepts ANC command 0x00 on the ANC server cluster                  |
+| ANC.S.C02.Tx  | Device generates ANC command response 0x02 on the ANC server cluster       |
+| ANC.S.E00     | Device generates ANC event 0x00 on the ANC server cluster                  |
+| ANC.C         | Device supports an ANC client                                              |
+| ANC.C.F00     | Device ANC client is capable of understanding the feature with bit 0       |
+| ANC.C.F01     | Device ANC client is capable of understanding the feature with bit 0       |
+| ANC.C.A0000   | Device ANC client is capable of reading or subscribing to attribute 0x0000 |
+| ANC.C.A0001   | Device ANC client is capable of reading or subscribing to attribute 0x0001 |
+| ANC.C.C00.Rsp | Device ANC client is capable of sending the command with id 0x00           |
+| ANC.C.C01.Rsp | Device ANC client is capable of sending the command with id 0x01           |
+| ANC.C.C02.Tx  | Device ANC client understands the command response with id 0x02            |
+| ANC.C.E00     | Device ANC client understands the event with id 0x00                       |
+
+More information about the PICS code format can be found at
+[PICS Guidelines](https://github.com/CHIP-Specifications/chip-test-plans/blob/master/docs/PICS%20Guidelines.md).
+
+In addition to these standard cluster PICS, other PICS may be defined to
+describe capabilities that are not directly expressed via the data model. For
+example, whether the device responds to manual operations that affect the Matter
+data model.
+
+Additionally, there are node-level PICS, which appear as a part of the MCORE
+PICS set. These PICS codes describe node level support such as the radio,
+whether the device is commissionable, whether the device comes with a QR code
+etc.
+
+PICS are used in testing in two ways. "Top level" PICS appear at the top of the
+test plan and indicate whether an entire test case should be run. For example, a
+test case for ANeatCluster would have a top-level PICS of ANC.S to indicate that
+the test would only be run if the ANC cluster is present on the endpoint. Test
+cases also use PICS to gate individual steps of the test which are not
+applicable if a certain element or capability is not implemented.
+
+The entire collection of PICS codes for a specification release is provided as a
+set of PICS XML files. These files are loaded into the PICS tool, which is used
+to manually set all the PICS codes for a device. The PICS XML files and the PICS
+tool are distributed as part of the official specification release package and
+are available on
+[Causeway](https://groups.csa-iot.org/wg/members-all/document/folder/2269).
```

Comment:
Since this is a public document, do we want to be clear that this requires a CSA member login?

Reply:
added wording that this is just for members.